src/fast-csv-wrapper.js

```diff
@@ -1,40 +1,66 @@
 import _ from 'lodash/fp'
 import { format as formatCSV } from '@fast-csv/format'
 import { keysToObject } from './futil'
+import { isIterable, isAsyncIterable } from './utils'
```

Comment:
At this point we should just roll our own thing. We might want an API like this:

```js
writeCSV({
  stream, // target stream
  iterableData, // iterator for each page of an array of objects
  headers: [{ field1: 'Label' }, 'fieldA', { field2: 'Label 1' }], // ordered list of fields and/or field:label pairs
  transformRecord, // function to transform each record 
  onWrite // function to intercept writing a page of records
})
```

Ultimately, contexture-export should just have an iterator and stream interface to the results